database/ddl/sqlite/worker.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package sqlite
+
+const (
+	// CreateWorkerTable represents a query to
+	// create the workers table for Vela.
+	CreateWorkerTable = `
+CREATE TABLE
+IF NOT EXISTS
+Workers (
+	id            INTEGER PRIMARY KEY AUTOINCREMENT,
+	hostname		TEXT,
+	address			TEXT,
+	routes          TEXT,
+	active          TEXT,
+	last_checked_in	INTEGER
+);
```

Comment:
Could we fix the spacing here?

Also the name of the table should be `workers` instead of `Workers` 👍 

```suggestion
CREATE TABLE
IF NOT EXISTS
workers (
	id              INTEGER PRIMARY KEY AUTOINCREMENT,
	hostname        TEXT,
	address         TEXT,
	routes          TEXT,
	active          TEXT,
	last_checked_in INTEGER
);
```